### DISCUSSION TITLE
[kdiff3] need  script for add to context menu

### DISCUSSION THREAD

**Discussion by shishirong** [ScoopInstaller/Scoop#6253] [Help and Questions]
how can I do it?

**Comment by HUMORCE** [comment]
Make a PR, if you know how to add it. otherwise create a manifest enhancement issue.

---

If you are talking about how to create the script: commonly, you will need to execute its original installer and record the registry changes. You can do this with the helpful tools like [Total Uninstall](https://www.martau.com/uninstaller-download.php)

**Comment by olcariz** [comment]
Hi guys, in previous installs, reg scripts were auto executed, how to enable it again now ?

**Reply by shishirong** [reply]
Usually the reg script can be found in the application directory, and you can execute it manually
